test/integration/requests.test.ts

```diff
@@ -316,6 +323,189 @@ describe("/requests OOO", function () {
     });
   });
 
+  describe.skip("PATCH /requests/:id", function () {
+    let testOooRequest;
+    let onboardingRequest;
+    let approvedOooRequest;
+    let rejectedOooRequest;
+
+    beforeEach(async function () {
+
+      oooRequestData = { ...createOooRequests3, userId: testUserId };
+      testOooRequest = await createRequest(oooRequestData);
```

Comment:
do we need the whole request data if not then we can just do this 
```suggestion
      testOooRequestId = await createRequest(oooRequestData).id;
```

Reply:
We can't do like this.
Thanks

Reply:
understood